canonical-websites/cn.ubuntu.com
# Update download links

## Done

Update download links

## QA

- Follow the instructions in README.md to get started
- Go to http://localhost:8010/download/ Make sure the 16.10 links have been replaced with 17.04 as per the doc
- Go to http://localhost:8010/desktop/ and make sure the laptop image has changed
## Issue / Card

### Docs 
- Download:  https://docs.google.com/document/d/1b8pXcsMPTzu5dXiG7qKBSvYzwV1aawQ12M4mzsOnQfM/edit#
- Desktop: https://docs.google.com/document/d/1NF1udWXWGmzw5gXCZYhPMWNmYQ3BCQ2t8QMRp5xb06c/edit#
